ansible/roles/test/files/ptftests/fast-reboot.py

```diff
@@ -310,24 +549,33 @@ def check_start(self):
     def iteration(self, is_stop):
         recorded_time = None
         counter = self.nr_tests
+        nr_from_upper_array = []
         while True:
-            success = self.ping_iteration()
+            success, nr_from_upper = self.ping_iteration()
+            nr_from_upper_array.append(nr_from_upper)
+            for _, q in self.ssh_jobs:
+                q.put('go')
             if success and is_stop or not success and not is_stop:
-                self.log("Success", True)
+                self.log("Base state", True)
```

Comment:
We're looking for a change in this function. So "Base state" it's a state which is expected before the change, "Changed" state it's a state which is expected after change.